modules/actor/actor-wfrp4e.js

```diff
@@ -1283,125 +1283,155 @@ export default class ActorWfrp4e extends WFRP4eDocumentMixin(Actor)
     }
   }
 
+  /**
+   * Decrements all injuries of this Actor
+   *
+   * @return {Promise<void>}
+   */
   async decrementInjuries() {
-    this.injuries.forEach(i => this.decrementInjury(i))
+    for (let i of this.injuries)
+      await this.decrementInjury(i);
   }
 
+  /**
+   * Decrements a `duration` value for the Injury.
+   * If `duration` reaches 0, inform about Injury being finished.
+   *
+   * @param {ItemWfrp4e}injury
+   *
+   * @return {Promise<*>}
+   */
   async decrementInjury(injury) {
     if (isNaN(injury.system.duration.value))
       return ui.notifications.notify(game.i18n.format("CHAT.InjuryError", { injury: injury.name }))
 
-    injury = duplicate(injury)
-    injury.system.duration.value--
+    injury = foundry.utils.duplicate(injury);
```

Comment:
Just in case there will be a time when Foundry will stop supporting global functions. They seem to push usage of the `foundry.utils` getter, and it makes for better IDE highlighting.

Reply:
They also seem to push `deepClone` instead of `duplicate` so that might as well change too.

Reply:
Changed